etl-pipeline/scripts/grin_mirror.py

```diff
@@ -0,0 +1,125 @@
+# An API interface to https://books.google.com/libraries/NYPL/,
+# per https://docs.google.com/document/d/1ugKUSkq4jAwmyWu3HubUIobQA1ag4VgRP1JjLeGUW20
```

Comment:
nit: we can take this out since we can't access it

Reply:
good catch - ill swap the links!